Source/player.cpp

```diff
@@ -1625,7 +1663,19 @@ void StartPlrHit(int pnum, int dam, bool forcehit)
 	if (!(plr[pnum]._pGFXLoad & PFILE_HIT)) {
 		LoadPlrGFX(pnum, PFILE_HIT);
 	}
-	NewPlrAnim(pnum, plr[pnum]._pHAnim[pd], plr[pnum]._pHFrames, 0, plr[pnum]._pHWidth);
+
+	int skippedAnimationFrames = 1; // Every GotHit start with Frame 2. Because ProcessPlayerAnimation is called after StartPlrHit and its increases the AnimationFrame.

```

Comment:
From my experience stomping around the hit-recovery animation code over the last few days, this comment isn't correct. I recommend referring to the discussion in #984.

Reply:
You are right. I fixed it.
Thank you very much. :)